code/modules/hydroponics/grafts.dm

```diff
@@ -0,0 +1,48 @@
+/*
+A new subsystem for hydroponics, as a way to share specific traits into plants, as a way to phase out the DNA manipulator.
+*/
+/obj/item/graft
+	name = "plant graft"
+	desc = "A carefully cut graft off of a freshly grown plant. Can be grafted onto a plant in order to share unique plant traits onto a plant."
+	w_class = WEIGHT_CLASS_TINY
+	icon = 'icons/obj/hydroponics/equipment.dmi'
+	icon_state = "graft_plant"
+	attack_verb = list("planted", "vegitized", "cropped", "reaped", "farmed")
+	///This stored the trait taken from the parent plant. Defaults to perenial growth.
+	var/datum/plant_gene/trait/stored_trait
+	///Determines the appearance of the graft. Rudimentary right now so it just picks randomly.
+	var/graft_appearance
+	///Seed type that the graft was taken from, used for applying parent stats.
+	var/obj/item/seeds/parent_seed = null
+
+/obj/item/graft/Initialize()
+	. = ..()
+	stored_trait = new /datum/plant_gene/trait/repeated_harvest //Default gene is repeated harvest.
+
+	graft_appearance = rand(100)
+	switch(graft_appearance)
+		if(0 to 25)
+			icon_state = "graft_plant"
+		if (26 to 50)
+			icon_state = "graft_flower"
+		if (51 to 95)
+			icon_state = "graft_mushroom"
+		if(96 to 100)
+			icon_state = "graft_doom"
```

Comment:
In case you are not using `graft_appearance` for other purposes you can just do a weighted `pick()`